### DISCUSSION TITLE
Site banner discussion

### DISCUSSION THREAD

**Discussion by techtolentino** [Qiskit/qiskit.org#2382] [General]
Opening a discussion around the Campaign Zero banner, and how/when we plan on removing the link to the campaign. 

For clarity:
This update does **not** mean that our team, or members of our community, no longer support this campaign. Rather, the issue with leaving the banner up (without a plan to remove it), defeats the deliberate decision to add it in the first place.

Hoping to hear others' thoughts on this.

**Comment by techtolentino** [comment]
Note: we'll be removing the BLM banner altogether.

**Comment by delapuente** [comment]
Following Slack conversations, we agree on the point you made.
